content/en/docs/configuring-sdk/_index.md

```diff
@@ -67,19 +67,250 @@ export AWS_REGION=us-west-2
 set AWS_REGION=us-west-2
 ```
 
-##### Specify Region Explicitly
+##### Specify Region Programmatically
 
 ```go
 cfg, err := config.LoadDefaultConfig(config.WithRegion("us-west-2"))
 ```
 
 ## Specifying Credentials
 
-The AWS SDK for Go requires credentials (an access key and secret access key) to sign requests to AWS. You can specify
-your credentials in several locations, depending on your particular use case.
+The {{% alias sdk-go %}} requires credentials (an access key and secret access
+key) to sign requests to AWS. You can specify your credentials in
+several locations, depending on your particular use case. For
+information about obtaining credentials, see [Getting Started]({{% relref "getting-started.md" %}}).
 
-When you initialize a new service client without providing any credential arguments, the SDK uses the default credential
-provider chain to find AWS credentials. The SDK uses the first provider in the chain that returns credentials without an
-error. The default provider chain looks for credentials in the following order: 
+When you initialize an `aws.Config` instance using `config.LoadDefaultConfig`,
```

Comment:
What do you think about using `apiref` for these instead of code blocks?

Reply:
My thought process was that when a page first introduces a reference like aws.Config or config.LoadDefaultConfig we link it to the API Reference guide. Then subsequent references are code blocks. This avoids having to manage a large number of links on the page and increasing the chance to get it wrong.